README.md

```diff
@@ -7,8 +7,57 @@ and can generate documentation in multiple formats including standard Javadoc, H
 
 ## Using dokka
 
+### Plugins 
+Dokka can be customized with plugins. Each output format (apart from the default `HTML` one is internally a plugin).
+Additionally, `kotlin-as-java` plugin can be used to generate documentation as seen from Java perspective. 
+Currently maintained plugins are:
+* `dokka-base` - the main plugin needed to run dokka, contains html format
+* `gfm-plugin` - configures `GFM` output format
+* `jekyll-plugin` - configures `Jekyll` output format
+* `javadoc-plugin` - configures `Javadoc` output format, automatically applies `kotlin-as-java-plugin` 
+* `kotlin-as-java-plugin` - translates Kotlin definitions to Java 
+
+Please see the usage instructions below for how to add plugins to dokka. 
+
+### Source sets 
+Dokka generates documentation based on source sets. 
+
+For single-platform projects, there is almost always only one source set - `main`.
+
+For multi-platform projects, source sets are the same as in Kotlin plugin - source set for each platform, eg. `jvmMain` or `jsMain` and for each
+common source set - eg. the default `commonMain` and custom ones, like `jsAndJvmMain`.
+When configuring multi-platform projects manually (eg. in the CLI), source sets must declare which other source sets they depend on. 
```

Comment:
I thought this is still also necessary for the gradle plugin?

Reply:
Yes, but only without autoconfiguration, I will clarify it